### PR TITLE
Add option: buildkite_agent_tags_from_gcp_labels

### PR DESCRIPTION
**Pull request by conorgil**
This adds support for the new buildkite agent configuration option introduced in https://github.com/buildkite/agent/pull/930.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `buildkite_agent_allow_service_startup` option.
 - `buildkite_agent_expose_secrets` option.
 - `buildkite_agent_start_parameters` option.
+- `buildkite_tags_from_gcp_labels` option.
```
Comment (petemounce) [inline CHANGELOG.md]:
```suggestion
- `buildkite_agent_tags_from_gcp_labels` option.
```

**Review by petemounce — COMMENTED** [review]
LGTM, just the changelog change. Thanks!

File: CHANGELOG.md (hunk 1 above)
Comment (conorgil) [reply CHANGELOG.md]: Good catch! Fixed.

**Comment by conorgil** [comment]
@petemounce Thanks for the feedback! Once this is merged, do you plan to update the CHANGELOG and cut a new release at `v1.2.0`?

**Comment by petemounce** [comment]
Of the ansible module? Yes, once @lox releases here.

**Review by petemounce — APPROVED** [review]

**Comment by conorgil** [comment]
@petemounce Now that this is merged, can you tag the repo at `1.2.0`? I'd like to update my Ansible `requirements.yml` file to pull down the correct version of this module.

**Comment by petemounce** [comment]
Done; apologies for the wait.

**Pull request merged**